src/domd_cfg.c

```diff
@@ -374,6 +374,13 @@ static uint32_t domd_irqs[] = {
 			185,
 };
 
+extern char __img_domd_start[];
+extern char __img_domd_end[];
+extern char __dtb_domd_start[];
+extern char __dtb_domd_end[];
+
+#define LOAD_ADDR_OFFSET 0x80000
```

Comment:
Commit description does not mention this. What is this offset and why it is equals to 0x80000?